### PR TITLE
Qa

### PR DESCRIPTION
**Pull request by agustinlg**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "api-designer",
-  "version": "0.2.0",
+  "version": "0.2.1",
```
Comment (llattes) [inline bower.json]: We should stick to 0.2.0, since Christian wants to release with this very particular release number.

**Comment by llattes** [comment]
Just to confirm, did you generate the `.min.js` files again, right? (Because I cannot see the diff)

**Comment by agustinlg** [comment]
Yes! On commit [028a09f6dac75b8a86cec06b5fba432e40416eb8](https://github.com/mulesoft/api-designer/pull/459/commits/028a09f6dac75b8a86cec06b5fba432e40416eb8)

**Comment by llattes** [comment]
👍 LGTM!

**Pull request merged**